en/concept/architecture.md

```diff
@@ -66,35 +73,39 @@ The message update rates can be inspected in real-time on the system by running
 `uorb top`.
 
 ## Runtime Environment
-PX4 runs on various Operating Systems that provide a POSIX-API and execution
+
+PX4 runs on various operating systems that provide a POSIX-API and execution
 model (such as Linux, MacOS, NuttX or QuRT).
 
 The inter-module communication (using uORB) is based on shared memory. The whole
 PX4 middleware runs in a single address space, i.e. memory is shared between all
-modules. However the system is designed such that with minimal effort it would
-be possible to run each module in separate address space (parts that would need
-to be changed include uORB, parameter interface, dataman and perf).
+modules. 
+
+> **Info** The system is designed such that with minimal effort it would
+> be possible to run each module in separate address space (parts that would need
+> to be changed include `uORB`, `parameter interface`, `dataman` and `perf`).
 
 There are 2 different ways that a module can be executed:
 - **Tasks**: this is the more common way. A module runs in its own task with its
   own stack and process priority.
-- **Work Queues**: the module runs on a shared task, meaning it does not own a
+- **Work queues**: the module runs on a shared task, meaning it does not own a
   stack. Multiple tasks run on the same stack with a single priority per work
   queue.
-  A task is scheduled by specifying a fixed time in the future.
-  The advantage is that it uses less RAM, but the task is not allowed to sleep
-  or poll on a message.
+  
+A task is scheduled by specifying a fixed time in the future.
```

Comment:
This is actually part of the work queue description, so I think it should be indented.